docs/overview.rst

```diff
@@ -29,50 +32,55 @@ The crash ingestion pipeline that we have at Mozilla looks like this:
       subgraph stores {
          rank=same;
 
-         aws [shape=tab, label="S3", style=filled, fillcolor=gray];
+         s3raw [shape=tab, label="S3 (Raw)", style=filled, fillcolor=gray];
+         pigeon [shape=cds, label="pigeon"];
+         rabbitmq [shape=tab, label="RMQ", style=filled, fillcolor=gray];
```

Comment:
Can we label this "RabbitMQ"? A google search for "RMQ" doesn't bring up RabbitMQ, so it's kinda hard to tell what this is if you're not familiar with RabbitMQ.